jasypt/src/main/java/org/jasypt/intf/service/JasyptStatelessService.java

```diff
@@ -56,6 +57,10 @@ public JasyptStatelessService() {
         super();
     }
     
+    private String randomString = RandomStringUtils.randomAlphanumeric(12);
+    private boolean displayed = false;
```

Comment:
@prakashiit  use more meaningful name like shoulPasswordBeDisplayed

Reply:
@SriMouli The significance of variable is this: Has the password been displayed yet? If false, display the password and set the value as true. If true, do nothing.
Please approve one of these names for this variable:
1. `hasPasswordBeenDisplayed`
2. `passwordDisplayed`
3. `shouldPasswordBeDisplayed`
4. `displayPassword`
(In cases 3 & 4, the default value would be true and will be changed to false after it is displayed). Any suggestion different from these four is also welcome.

Reply:
Done!